rpc/client.go

```diff
@@ -43,7 +43,7 @@ type response[T serde.ResponsePayload] struct {
 }
 
 // NewRpcClient creates a new RpcClient
-func NewRpcClient(rpcServerUrl string, logger zerolog.Logger, trans transport.Requester) (*RpcClient, error) {
+func NewRpcClient(logger zerolog.Logger, trans transport.Requester) (*RpcClient, error) {
```

Comment:
This is just a removal of an unused parameter. The url in question has already been consumed by the `transport.Requester` parameter.